packages/ui/src/SmartResourcePropertyDisplay.tsx

```diff
@@ -0,0 +1,19 @@
+import { PropertyType } from '@medplum/core';
+import { Resource } from '@medplum/fhirtypes';
+import { evalFhirPath } from '@medplum/fhirpath';
+import React from 'react';
+import { ResourcePropertyDisplay } from '.';
+
+export interface SmartResourcePropertyDisplayProps {
+  resource: Resource;
+  path: string;
+}
+
+export function SmartResourcePropertyDisplay(props: SmartResourcePropertyDisplayProps): JSX.Element {
+  const value = evalFhirPath(props.path, props.resource);
+
+  if (value?.[0] && value.length === 1) {
+    return <ResourcePropertyDisplay value={value?.[0] || ''} propertyType={PropertyType.string} />;
```

Comment:
You should be able to just do `value[0]`

Thoughts on making `propertyType` an input prop to the overall component?

Reply:
> Thoughts on making propertyType an input prop to the overall component?

sgtm